ambari/install/getting-ready/system-requirements.rst

```diff
@@ -0,0 +1,167 @@
+
+::
+
+	free -m
+
+If you plan to install the Ambari Metrics Service (AMS) into your cluster the host you plan to run the Ambari Metrics Collector host should have the following memory and disk space available based on cluster size:
+
++-----------------+------------------+------------+
+| Number of hosts | Memory Available | Disk Space |
++=================+==================+============+
+| 1               | 1024 MB          | 10 GB      |
++-----------------+------------------+------------+
+| 10              | 1024 MB          | 20 GB      |
++-----------------+------------------+------------+
+| 50              | 2048 MB          | 50 GB      |
++-----------------+------------------+------------+
+| 100             | 4096 MB          | 100 GB     |
++-----------------+------------------+------------+
+| 300             | 4096 MB          | 100 GB     |
++-----------------+------------------+------------+
+| 500             | 8096 MB          | 200 GB     |
++-----------------+------------------+------------+
+| 1000            | 12288 MB         | 200 GB     |
++-----------------+------------------+------------+
+| 2000            | 16348 MB         | 500 GB     |
++-----------------+------------------+------------+
+
+
+.. note::
+  The above is offered as guidelines. Be sure to test for your particular environment. Also refer to :ref:`Package Size and Inode Count Requirements<ambari-inode-count>` for more information on package size and Inode counts.
+
+.. _ambari-inode-count:
+
+Package Size and Inode Count Requirements
+_________________________________________
+
+Size and Inode values are approximate
+
++----------------------------+----------+-----------+
+|                            | Size     | Inodes    |
++============================+==========+===========+
+| Ambari Server              | 100MB    | 5000      |
++----------------------------+----------+-----------+
+| Ambari Agent               | 8MB      | 1000      |
++----------------------------+----------+-----------+
+| Ambari Metrics Collector   | 225MB    | 4000      |
++----------------------------+----------+-----------+
+| Ambari Metrics Monitor     | 1MB      | 100       |
++----------------------------+----------+-----------+
+| Ambari Metrics Hadoop Sink | 8MB      | 100       |
++----------------------------+----------+-----------+
+| After Ambari Server Setup  | N/A      | 4000      |
++----------------------------+----------+-----------+
+| After Ambari Server Start  | N/A      | 500       |
++----------------------------+----------+-----------+
+| After Ambari Agent Start   | N/A      | 200       |
++----------------------------+----------+-----------+
+
+Check the Maximum Open File Descriptors
+_______________________________________
+
+The recommended maximum number of open file descriptors is 10000, or more. To check the current value set for the maximum number of open file descriptors, execute the following shell commands on each host:
```

Comment:
Using a divider could increase readability (be advised that in US English the standard divider is the comma).